sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/LocalRelation.scala

```diff
@@ -67,6 +77,8 @@ case class LocalRelation(
   // A local relation must have resolved output.
   require(output.forall(_.resolved), "Unresolved attributes found when constructing LocalRelation.")
 
+  var isSqlScript: Boolean = false
```

Comment:
this is very hacky...

Reply:
It is a temporary solution until SQL Scripting is fully integrated with Spark Connect. We should see what @hvanhovell has to say about this implementation, but in general it was a part of our agreement to do it this way.